### PR TITLE
Added Translations for Backend

### PR DESCRIPTION
**Pull request by hefan**
Product Screens, Promotions, User, Hints

### CODE REVIEW

**Comment by kennyadsl** [comment]
@hefan can you please rebase? 

@tvdeyen can you please take a look? 🇩🇪

**Comment by hefan** [comment]
ok, its rebased

File: config/locales/de.yml [hunk 1]
```diff
@@ -577,8 +588,86 @@ de:
     adjustment_successfully_closed: Anpassung wurde erfolgreich geschlossen!
     adjustment_successfully_opened: Anpassung wurde erfolgreich geöffnet!
     adjustment_total: Anpassungen Gesamt
+    adjustment_types: Anpassungensarten
     adjustments: Anpassungen
     admin:
+      images:
+        index:
+          choose_files: Wähle Dateien zum hochladen aus
+          drag_and_drop: oder per drag and drop hier
```
[the remark this answers is not in the document]
Comment (hefan) [reply config/locales/de.yml]: erledigt

File: config/locales/de.yml [hunk 2]
```diff
@@ -577,8 +588,86 @@ de:
     adjustment_successfully_closed: Anpassung wurde erfolgreich geschlossen!
     adjustment_successfully_opened: Anpassung wurde erfolgreich geöffnet!
     adjustment_total: Anpassungen Gesamt
+    adjustment_types: Anpassungensarten
     adjustments: Anpassungen
     admin:
+      images:
+        index:
+          choose_files: Wähle Dateien zum hochladen aus
```
[the remark this answers is not in the document]
Comment (hefan) [reply config/locales/de.yml]: ja. finde ich auch

File: config/locales/de.yml [hunk 3]
```diff
@@ -577,8 +588,86 @@ de:
...
+          amount_less_than: Betrag kleiner als
+          new_price: Neuer Preis
+        new:
+          new_price: Neuer Preis
+      promotions:
+        actions:
+          calculator_label: Berechnet von
+        activations_edit:
+          auto: Alle Bestellungen werden versuchen diese Werbeaktion zu benutzen.
+          multiple_codes_html: Diese Werbeaktion benutzt %{count} promotion codes
```
[the remark this answers is not in the document]
Comment (hefan) [reply config/locales/de.yml]: Erledigt

File: config/locales/de.yml [hunk 4]
```diff
@@ -577,8 +588,86 @@ de:
...
+        credit_type: Typ
+        current_balance: 'Aktuelles Guthaben:'
+        edit: Guthaben bearbeiten
+        edit_amount: Guthabenbetrag bearbeiten
+        errors:
+          amount_authorized_exceeds_total_credit: " überschreitet das erhältliche Guthaben"
+          amount_used_cannot_be_greater: kann nicht größer als der gutgeschriebene Betrag sein
+          amount_used_not_zero: ist grüßer als Null. Guthaben kann nicht gelöscht werden.
+          cannot_be_modified: kann nicht geändert werden
+          cannot_change_used_store_credit: Guthaben, dass beasnsprucht wurde, kann nicht geändert werden
```
[the remark this answers is not in the document]
Comment (hefan) [reply config/locales/de.yml]: behoben

File: config/locales/de.yml [hunk 5]
```diff
@@ -577,8 +588,86 @@ de:
...
+        invalidated: ungültig erklärt
+        issued_on: Ausgestellt am
+        memo: Memo
+        new: Neues Guthaben
+        no_store_credit_selected: Kein Guthaben wurde ausgewählt
+        payment_originator: 'Zahlung - Bestellung #%{order_number}'
+        reason_for_updating: Grund für Aktualisierung
+        refund_originator: 'Rückerstatten - Bestellung #%{order_number}'
+        resource_name: Guthaben
+        select_amount_store_credit_reason: Gib einen Grund an um die Summe zu aktualisieren
```
[the remark this answers is not in the document]
Comment (hefan) [reply config/locales/de.yml]: behoben

File: config/locales/de.yml [hunk 6]
```diff
@@ -577,8 +588,86 @@ de:
...
+        select_amount_update_reason: Gib einen Grund an um die Summe zu aktualisieren
+        select_reason: Gib einen Grund für dieses Guthaben an
+        total_unused: Gesamt unbenutzt
+        type_html_header: Kreditart
+        unable_to_create: Guthaben konnte nciht angelegt werden
```
[the remark this answers is not in the document]
Comment (hefan) [reply config/locales/de.yml]: behoben

File: config/locales/de.yml [hunk 7]
```diff
@@ -577,8 +588,86 @@ de:
...
+        issued_on: Ausgestellt am
+        memo: Memo
+        new: Neues Guthaben
+        no_store_credit_selected: Kein Guthaben wurde ausgewählt
+        payment_originator: 'Zahlung - Bestellung #%{order_number}'
+        reason_for_updating: Grund für Aktualisierung
+        refund_originator: 'Rückerstatten - Bestellung #%{order_number}'
+        resource_name: Guthaben
+        select_amount_store_credit_reason: Gib einen Grund an um die Summe zu aktualisieren
+        select_amount_update_reason: Gib einen Grund an um die Summe zu aktualisieren
```
[the remark this answers is not in the document]
Comment (hefan) [reply config/locales/de.yml]: behoben

**Comment by hefan** [comment]
ok, i have checked all capitalization issues and typos and fixed them

**Comment by kennyadsl** [comment]
@hefan Hey there! Can you please squash commits into a single one?

**Comment by hefan** [comment]
ok. last 3 commits squashed

**Comment by kennyadsl** [comment]
Thanks @hefan!

**Pull request merged**